### PR TITLE
Neuron observability

### PR DESCRIPTION
**Pull request by freschri**
Single New EKS Cluster Open Source Observability Accelerator for Neuron-based clusters
depends on https://github.com/aws-observability/aws-observability-accelerator/pull/32
@elamaran11 please check if we want to keep/remove cdk.json from gitignore (we need to be able to add entries that are validated in compilation otherwise github compilation fails on push)

### CODE REVIEW

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# Single New EKS Cluster Open Source Observability Accelerator for Neuron-based clusters
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:1]: Rename this to `Single Cluster Open Source Observability - Neuron-based cluster`

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+Amazon EC2 ML instances belong to the [Trn1/Trn1n](https://aws.amazon.com/ec2/instance-types/trn1/), [Inf1](https://aws.amazon.com/ec2/instance-types/inf1/) and [Inf2](https://aws.amazon.com/ec2/instance-types/inf2/) families. Trn1/Trn1n instances feature multiple AWS Trainium accelerators and support high-performance training. Inf1 and Inf2 instances feature multiple AWS Inferentia accelerators and support high-performance and low-latency inference.
+
+[AWS Neuron](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/) is an SDK with a compiler, runtime, and profiling tools that helps developers deploy models on both AWS Inferentia accelerators and train them on AWS Trainium accelerators. It integrates natively with popular ML frameworks, such as PyTorch and TensorFlow.
+
+This pattern shows you how to monitor the performance of ML accelerators, used in an Amazon EKS cluster leveraging Inferentia-based Amazon EC2 Inf1 instances. The pattern will also be applicable to Inf2 and Trn1/Trn1n instances soon, when AWS Cloud Development Kit (CDK) completes support for Inf2 and Trn1/Trn1n (in progress, at the time of writing).
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:9]: rename accelerators to workloads

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+Amazon Managed Grafana is a managed service for Grafana, a popular open-source analytics platform that enables you to query, visualize, and alert on your metrics, logs, and traces.
+
+## Objective
+
+This pattern deploys an Amazon EKS cluster with a node group that includes Inf1 instances.
+
+The AMI type of the node group is `AL2_x86_64_GPU AMI`, which uses the [Amazon EKS-optimized accelerated AMI](https://aws.amazon.com/marketplace/pp/prodview-nwwwodawoxndm). In addition to the standard Amazon EKS-optimized AMI configuration, the accelerated AMI includes the AWS Neuron container runtime.
+
+The Neuron container runtime consists of kernel driver and C/C++ libraries which provide APIs to access Inferentia and Trainium Neuron devices. The Neuron ML frameworks plugins for TensorFlow, PyTorch and Apache MXNet use the Neuron runtime to load and run models on the NeuronCores.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:23]: NeuronCores -> Neuron Cores

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+Neuron metrics are exposed to Amazon Managed Service for Prometheus by the `neuron-monitor` DaemonSet, which deploys a minimal container, with the [Neuron Tools](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/tools/index.html) installed. Specifically, the container runs the [`neuron-monitor`](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/tools/neuron-sys-tools/neuron-monitor-user-guide.html#neuron-monitor-ug) command piped into the `neuron-monitor-prometheus.py` companion script:
+
+```bash
+neuron-monitor | neuron-monitor-prometheus.py --port <port>
+```
+
+`neuron-monitor` collects metrics and stats from the Neuron Applications running on the system and streams the collected data to stdout [in JSON format](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/tools/neuron-sys-tools/neuron-monitor-user-guide.html#the-json-output-format).
+
+`neuron-monitor-prometheus.py` maps and exposes the telemetry data from JSON format into Prometheus compatible format.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:35]: where is `neuron-monitor-prometheus.py` ?

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+        "GRAFANA_KUBELET_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/kubelet.json",
+        "GRAFANA_NSWRKLDS_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/namespace-workloads.json",
+        "GRAFANA_NODEEXP_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/nodeexporter-nodes.json",
+        "GRAFANA_NODES_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/nodes.json",
+        "GRAFANA_WORKLOADS_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/workloads.json",
+        "GRAFANA_NEURON_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/neuron/neuron-monitor.json"
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:123]: Dont you need to add this dash to `aws-observability-accelerato`r repo?

File: lib/single-new-eks-opensource-observability-pattern/neuron/neuron-device-plugin-addon.ts [hunk 6]
```diff
@@ -0,0 +1,44 @@
+import { KubernetesManifest, ServiceAccount } from "aws-cdk-lib/aws-eks";
```
Comment (elamaran11) [inline lib/single-new-eks-opensource-observability-pattern/neuron/neuron-device-plugin-addon.ts:1]: You should move this to blueprints repo. I see an active PR - https://github.com/aws-quickstart/cdk-eks-blueprints/pull/777 for this. Lets get this from blueprints vs using it here. Having in blueprints make this available to all.

File: lib/single-new-eks-opensource-observability-pattern/neuron/neuron-device-plugin-role.yaml [hunk 7]
```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```
Comment (elamaran11) [inline lib/single-new-eks-opensource-observability-pattern/neuron/neuron-device-plugin-role.yaml:2]: this should also move to blueprints repo.

File: lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor-addon.ts [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import { KubernetesManifest } from "aws-cdk-lib/aws-eks";
```
Comment (elamaran11) [inline lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor-addon.ts:1]: This should also move to blueprints

File: lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor.yaml [hunk 9]
```diff
@@ -0,0 +1,85 @@
+kind: DaemonSet
```
Comment (elamaran11) [inline lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor.yaml:1]: This should also move to blueprints

File: lib/common/resources/neuron/pytorch-inference-resnet50.yml [hunk 10]
```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
```
Comment (elamaran11) [inline lib/common/resources/neuron/pytorch-inference-resnet50.yml:1]: This file should be moved to aws-observability-accelerator-repo and i would prefer using a GitOps approach to deploy the workload

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 11]
```diff
@@ -0,0 +1,268 @@
...
+
+[AWS Trainium](https://aws.amazon.com/machine-learning/trainium/) and [AWS Inferentia](https://aws.amazon.com/machine-learning/inferentia/) are accelerated Machine Learning (ML) chips (or ML accelerators), designed and built by AWS. They are also referred to as Neuron Devices. Each Neuron device includes multiple NeuronCores, the machine learning compute cores.
+
+Amazon EC2 ML instances belong to the [Trn1/Trn1n](https://aws.amazon.com/ec2/instance-types/trn1/), [Inf1](https://aws.amazon.com/ec2/instance-types/inf1/) and [Inf2](https://aws.amazon.com/ec2/instance-types/inf2/) families. Trn1/Trn1n instances feature multiple AWS Trainium accelerators and support high-performance training. Inf1 and Inf2 instances feature multiple AWS Inferentia accelerators and support high-performance and low-latency inference.
+
+[AWS Neuron](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/) is an SDK with a compiler, runtime, and profiling tools that helps developers deploy models on both AWS Inferentia accelerators and train them on AWS Trainium accelerators. It integrates natively with popular ML frameworks, such as PyTorch and TensorFlow.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:7]: There term accelerators is unclear here

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md [hunk 12]
```diff
@@ -0,0 +1,268 @@
...
+**Note**: insure your selected instance type is available in your selected region. To check that, you can run the following command (amend `Values` below as you see fit):
+
+```bash
+aws ec2 describe-instance-type-offerings \
+    --filters Name=instance-type,Values="inf1*" \
+    --query "InstanceTypeOfferings[].InstanceType" \
+    --region us-east-2
+```
+
+8. For the `neuron-monitor` DaemonSet, you can either use the image already referenced into the manifest, or build your own using the Dockerfile at location `cdk-aws-observability-accelerator/lib/common/resources/neuron/neuron-monitor.dockerfile`, [push it to an ECR repository](https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html) of your choice and update the image URL in the manifest at location `cdk-aws-observability-accelerator/lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor.yaml`.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:154]: `neuron-monitor` should be moved to blueprints addon. You can move the `neuron-monitor.dockerfile` to `aws-observability-accelerator` repo and you can show in blueprints documentation on how to build the docker file with addon document.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@freschri This is AWSome work. We need some strategic and tactical changes as per the comments below to align this well. Love the neuron pattern.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 2 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:9]: that's their name.. workloads is not right in this context. e.g. see here https://aws.amazon.com/machine-learning/neuron/ "AWS Trainium and AWS Inferentia accelerators "

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 3 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:23]: that's the name, please see here: https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/neuron-hardware/neuroncores-arch.html

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 4 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:35]: it is explained a few lines above, "Specifically, the container runs the neuron-monitor command piped into the `neuron-monitor-prometheus.py` companion script", so it is inside the container

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 5 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:123]: yes I mentioned the dependency in the description of this pr: "depends on https://github.com/aws-observability/aws-observability-accelerator/pull/32"

File: lib/common/resources/neuron/pytorch-inference-resnet50.yml (hunk 10 above)
Comment (freschri) [reply lib/common/resources/neuron/pytorch-inference-resnet50.yml:1]: it is an example workload, just generating metrics, not a real one never to be used in production.
so I believe deployed manually as a pod is better?

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 11 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md:7]: it is explained at the beginning of the doc: "[AWS Trainium]and [AWS Inferentia] are accelerated Machine Learning (ML) chips (or ML accelerators)", see also here: https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/glossary.html?highlight=glossary#neuron-devices-accelerated-machine-learning-chips

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 1 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md]: done

**Comment by elamaran11** [comment]
@freschri Any updates to this PR?

**Comment by freschri** [comment]
> @freschri Any updates to this PR?

I addressed all points apart from the addons. One of them depends on another PR in Blueprints (Neuron Device Plugin Addon). My suggestion was on another channel to merge this PR as is, if no other concern, and raise 2 issues to port the addons to Blueprints. That can be done as an exercise by someone with limited experience on Blueprints/Patterns/Observability. Please let me know

**Comment by elamaran11** [comment]
> > @freschri Any updates to this PR?
> 
> I addressed all points apart from the addons. One of them depends on another PR in Blueprints (Neuron Device Plugin Addon). My suggestion was on another channel to merge this PR as is, if no other concern, and raise 2 issues to port the addons to Blueprints. That can be done as an exercise by someone with limited experience on Blueprints/Patterns/Observability. Please let me know

Thats a good opportunity mentor, if you have some one in your region. If i have someone, i will let you know. But this is not a burning one, please take it slow.

**Comment by ariveroi** [comment]
New pr against inf1 branch with new quickstart version: https://github.com/aws-observability/cdk-aws-observability-accelerator/pull/153

If we merge #153 PR to inf1 branch we will be able to merge this PR

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md (hunk 12 above)
Comment (freschri) [reply docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md]: done

File: lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor-addon.ts (hunk 8 above)
Comment (freschri) [reply lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor-addon.ts]: done

File: lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor.yaml (hunk 9 above)
Comment (freschri) [reply lib/single-new-eks-opensource-observability-pattern/neuron/neuron-monitor.yaml]: done

File: lib/common/resources/otel-collector-config-new.yml [hunk 13]
```diff
@@ -0,0 +1,1622 @@
+#
```
Comment (elamaran11) [inline lib/common/resources/otel-collector-config-new.yml:1]: Remove this file, should not be checked in!

File: lib/common/resources/neuron/pytorch-inference-resnet50.yml (hunk 10 above)
Comment (elamaran11) [reply lib/common/resources/neuron/pytorch-inference-resnet50.yml:1]: I would say deploy this manually as a pattern. DockerFile should be moved to a different repo may be `aws-observability-accelerator-repo `

File: lib/common/resources/neuron/pytorch-inference-resnet50.yml [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+kind: ConfigMap
```
Comment (elamaran11) [inline lib/common/resources/neuron/pytorch-inference-resnet50.yml:2]: Move this file outside of repo!

File: .gitignore [hunk 15]
```diff
@@ -12,7 +12,6 @@ node_modules
 # CDK asset staging directory
 .cdk.staging
 cdk.out
-cdk.json
```
Comment (elamaran11) [inline .gitignore:15]: Why is this removed?

File: bin/single-new-eks-inferentia-opensource-observability.ts [hunk 16]
```diff
@@ -0,0 +1,6 @@
+import SingleNewEksInferentiaOpenSourceObservabilityPattern from '../lib/single-new-eks-opensource-observability-pattern/neuron/inferentia-index';
+import { configureApp } from '../lib/common/construct-utils';
+
+const app = configureApp();
+
+new SingleNewEksInferentiaOpenSourceObservabilityPattern(app, 'single-new-eks-inferentia-opensource');
```
Comment (elamaran11) [inline bin/single-new-eks-inferentia-opensource-observability.ts:6]: Update the `readMe.md` `make list` command output with this new pattern.

File: lib/common/resources/neuron/pytorch-inference-resnet50.yml (hunk 14 above)
Comment (freschri) [reply lib/common/resources/neuron/pytorch-inference-resnet50.yml:2]: would this be a good location: https://github.com/aws-observability/aws-observability-accelerator/tree/main/artifacts/k8s-deployment-manifest-templates ?

Comment (elamaran11) [reply lib/common/resources/neuron/pytorch-inference-resnet50.yml:2]: Works for me!

Comment (elamaran11) [reply lib/common/resources/neuron/pytorch-inference-resnet50.yml:2]: create a new folder like `neuron`

Comment (freschri) [reply lib/common/resources/neuron/pytorch-inference-resnet50.yml:2]: done, see pr https://github.com/aws-observability/aws-observability-accelerator/pull/39

File: bin/single-new-eks-inferentia-opensource-observability.ts (hunk 16 above)
Comment (freschri) [reply bin/single-new-eks-inferentia-opensource-observability.ts:6]: it is there

File: .gitignore (hunk 15 above)
Comment (freschri) [reply .gitignore]: done

File: lib/common/resources/otel-collector-config-new.yml (hunk 13 above)
Comment (freschri) [reply lib/common/resources/otel-collector-config-new.yml]: done

File: cdk.json [hunk 17]
```diff
@@ -32,21 +32,33 @@
         "GRAFANA_NSWRKLDS_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/namespace-workloads.json",
         "GRAFANA_NODEEXP_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/nodeexporter-nodes.json",
         "GRAFANA_NODES_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/nodes.json",
-        "GRAFANA_WORKLOADS_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/workloads.json"
+        "GRAFANA_WORKLOADS_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/infrastructure/workloads.json",
+        "GRAFANA_NEURON_DASH_URL" : "https://raw.githubusercontent.com/aws-observability/aws-observability-accelerator/main/artifacts/grafana-dashboards/eks/neuron/neuron-monitor.json"
```
Comment (elamaran11) [inline cdk.json:36]: Remove this from `cdk.json` this should go in to doc only. We dont add the extra boards for all patterns.

File: lib/single-new-eks-fargate-opensource-observability-pattern/index.ts [hunk 18]
```diff
@@ -122,8 +122,7 @@ export default class SingleNewEksFargateOpenSourceObservabilityConstruct {
         Reflect.defineMetadata("ordered", true, blueprints.addons.GrafanaOperatorAddon);
         const addOns: Array<blueprints.ClusterAddOn> = [
             new blueprints.addons.VpcCniAddOn(),
-            new blueprints.addons.CoreDnsAddOn({
-                version: "v1.10.1-eksbuild.6",
+            new blueprints.addons.CoreDnsAddOn("v1.10.1-eksbuild.6", {
```
Comment (elamaran11) [inline lib/single-new-eks-fargate-opensource-observability-pattern/index.ts:125]: version can be populated with auto or can be removed!

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@freschri Couple of minor ones. Once you fix these, i can run e2e.

File: lib/single-new-eks-fargate-opensource-observability-pattern/index.ts (hunk 18 above)
Comment (freschri) [reply lib/single-new-eks-fargate-opensource-observability-pattern/index.ts]: done

File: cdk.json (hunk 17 above)
Comment (freschri) [reply cdk.json]: done

File: package.json [hunk 19]
```diff
@@ -24,16 +24,16 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
-    "@aws-quickstart/eks-blueprints": "^1.13.1",
-    "aws-cdk": "2.114.1",
+    "@aws-quickstart/eks-blueprints": "^1.14.0",
+    "aws-cdk": "2.130.0",
```
Comment (elamaran11) [inline package.json:28]: "aws-cdk-lib": "2.132.0",
        "aws-cdk": "2.132.0"

**Review by elamaran11 — CHANGES_REQUESTED** [review]
Update the CDK version.

```
        "aws-cdk-lib": "2.132.0",
        "aws-cdk": "2.132.0"
```

**Comment by elamaran11** [comment]
/do-e2e-test single-new-eks-inferentia-opensource-observability deploy

File: cdk.json [hunk 20]
```diff
@@ -37,17 +37,28 @@
       "kustomizations": [
         {
           "kustomizationPath": "./artifacts/grafana-operator-manifests/eks/infrastructure"
+        },
```
Comment (elamaran11) [inline cdk.json:40]: Remove all neuron references from `cdk.json`. `cdk.json` is placeholder for generic pattern.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
One more feedback

**Comment by freschri** [comment]
@elamaran11 updated blueprints version by merging main into here. please trigger e2e thanks

**Comment by elamaran11** [comment]
/do-e2e-test single-new-eks-inferentia-opensource-observability deploy

**Comment by freschri** [comment]
@elamaran11 e2e passes. shall we merge?

**Comment by elamaran11** [comment]
> @elamaran11 e2e passes. shall we merge?

Nope lets wait, I might to crash e2e and do few other checks.

**Comment by elamaran11** [comment]
The E2E Failure is related to subnet ENI which is very interim. It worked on retry,

**Comment by elamaran11** [comment]
/do-e2e-test single-new-eks-inferentia-opensource-observability deploy

**Comment by elamaran11** [comment]
/do-e2e-test single-new-eks-inferentia-opensource-observability destroy

**Review by elamaran11 — APPROVED** [review]
LGTM
